src/blockchain.js

```diff
@@ -44,12 +50,20 @@ Blockchain.connect = function(connectionList, opts, doneCb) {
           this.blockchainConnector.setProvider(ethereum);
         }
         return checkConnect(next);
-      } catch (error) {
-        return next(null, false);
+      } catch (err) {
+        return next(null, {
+          connectionString: 'web3://',
```

Comment:
why not an empty string?
If you keep the name `error`
you can pass it in the hash with: 
`{ error }`

Reply:
I don't want to use an empty string because we want some sort of context when an error gets raised.

Cool trick on keeping the name! 👍